vmm/src/acpi.rs

```diff
@@ -34,6 +34,7 @@ pub const ACPI_APIC_GENERIC_REDISTRIBUTOR: u8 = 14;
 #[cfg(target_arch = "aarch64")]
 pub const ACPI_APIC_GENERIC_TRANSLATOR: u8 = 15;
 
+#[allow(dead_code)]
```

Comment:
@likebreath 
Ah I realized after I merged this PR that you could have specified the `#[allow(dead_code)]` for the specific field that is not used.
Could you send another PR to fix that? This way it's clearer what is not being used by the code, rather than tagging the entire structure as not being used.